io.github.nemuelw.proton-desktop.metainfo.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Copyright 2024 Nemuel Wainaina -->
+<component type="desktop-application">
+  <id>io.github.nemuelw.proton-desktop</id>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <name>Proton Desktop</name>
```

Comment:
You cannot use proton as your application name, that's Proton's brand name

Reply:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#no-trademark-violations